packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -126,59 +131,74 @@ The `status` prop can be used to indicate the status of each item. The following
-		{ label: 'Organisations', status: 'started', onClick: console.log },
-		{ label: 'Business contacts', status: 'error', onClick: console.log },
-		{ label: 'Case studies', status: 'todo', onClick: console.log },
+		{ href: '#introduction', label: 'Introduction', status: 'done' },
+		{ href: '#organisations', label: 'Organisations', status: 'started' },
 		{
-			label: 'Attachments',
+			href: '#contacts',
+			label: 'Business contacts',
 			status: 'started',
-			onClick: console.log,
-			isActive: true,
 		},
-		{ label: 'Review and submit', status: 'blocked', onClick: console.log },
+		{ href: '#case-studies', label: 'Case studies', status: 'todo' },
+		{ href: '#attachments', label: 'Attachments', status: 'blocked' },
 	]}
 />
 ```
 
-## Hiding the sub title
+## Adding level 2 steps
 
-The Progress indicator component displays a subtitle "x of y steps completed" by default. To hide this subtitle, set the `hideSubtitle` prop to `true`.
+If a level 1 page has a secondary step, such as editing its content, use a separate page for this secondary step.
+
+When users navigate to this secondary step, let them know where they are by adding a nested level 2 step `item`. Only one level 2 step may exist for each level 1 step.
 
 ```jsx live
 <ProgressIndicator
+	activePath="#organisations/add-organisation"
 	hideSubtitle
 	items={[
-		{ href: '#', label: 'Introduction', status: 'done' },
-		{ href: '#', label: 'Organisations', status: 'started' },
+		{ href: '#introduction', label: 'Introduction', status: 'done' },
 		{
-			href: '#',
-			label: 'Business contacts',
+			href: '#organisations',
+			label: 'Organisations',
 			status: 'started',
-			isActive: true,
+			items: [
+				{
+					label: 'Add an organisation',
+					href: '#organisations/add-organisation',
+				},
+			],
 		},
-		{ href: '#', label: 'Case studies', status: 'todo' },
-		{ href: '#', label: 'Attachments', status: 'blocked' },
+		{ href: '#review', label: 'Review and submit', status: 'blocked' },
 	]}
 />
 ```
 
+### An example of a level 2 step
+
+Progress indicators contain level 1 links to each page of a multi-step form. When additional pages are required for secondary actions nested in level 1 pages, level 2 links are revealed in the Progress indicator to help a user understand where they have navigated to and how to return to the to the previous step.
+
+<ImageWithBorder src="/img/progress-indicator-level-two--1.png" alt="" />
+
+The level 2 step is separate page, so it is displayed under the main level 1 step in the progress indicator. This helps the user understand where they are in the process.
+
+<ImageWithBorder src="/img/progress-indicator-level-two--2.png" alt="" />
+
+Once the user saves their changes, they’re taken back to the main step page and a section alert is displayed to let them know that their changes were made successfully. The level 2 step is no longer displayed in the progress indicator. It’s only displayed when the user is on the level 2 step page.
```

Comment:
We seem to always use a capital P elsewhere
```suggestion
Once the user saves their changes, they’re taken back to the main step page and a section alert is displayed to let them know that their changes were made successfully. The level 2 step is no longer displayed in the Progress indicator. It’s only displayed when the user is on the level 2 step page.
```

Reply:
Not everywhere 😅